core/block_rendering_rewrite/measurables.js

```diff
@@ -208,6 +263,86 @@ Blockly.BlockRendering.ExternalValueInput = function(input) {
 goog.inherits(Blockly.BlockRendering.ExternalValueInput,
     Blockly.BlockRendering.Input);
 
+/**
+ * An object containing information about the space a previous connection takes
+ * up during rendering.
+ * @package
+ * @constructor
+ */
+Blockly.BlockRendering.PreviousConnection = function() {
+  Blockly.BlockRendering.PreviousConnection.superClass_.constructor.call(this);
+  this.renderRect = null;
+  this.type = 'previous connection';
+  this.height = BRC.MEDIUM_PADDING;
```

Comment:
Add a new constant for this height.  `BRC.STACK_CONNECTION_HEIGHT`, maybe?  Use it here and for NextConnection, and set it equal to `BRC.MEDIUM_PADDING`.  Changing connection height/shapes is a common tweak for rendering.

Reply:
Or `NOTCH_HEIGHT, since we seem to be calling those 'notches'